great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -1,10 +1,14 @@
+import logging
 import os
 import random
 import re
+<<<<<<< HEAD
+# PYTHON 2 - py2 - update to ABC direct use rather than __metaclass__ once we drop py2 support
```

Comment:
Address merge conflict

Reply:
We were pair walking though this. Resolved.